### PR TITLE
activemq: Adding plist again, addressing #39477

### PR DESCRIPTION
**Pull request by douglasjose**
- [:white_check_mark:] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [:white_check_mark:] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [:white_check_mark:] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [:white_check_mark:] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

activemq: Adding plist configured with `AbandonProcessGroup=true`, addressing original issue in #39477.

### CODE REVIEW

**Comment by dunn** [comment]
Thanks! Merged as 6cafb22903e371d48b370033085912750f4f6ff4.

File: Formula/activemq.rb [hunk 1]
```diff
@@ -14,6 +14,32 @@ def install
...
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>AbandonProcessGroup</key>
+        <true/>
+        <key>WorkingDirectory</key>
+        <string>#{libexec}</string>
```
Comment (DomT4) [inline Formula/activemq.rb]: This should have probably been `opt_libexec`, or we're going to have the same problem where the plist needs unloading, replacing & reloading every revision/update.

Comment (DomT4) [inline Formula/activemq.rb]: https://github.com/Homebrew/homebrew-core/commit/7af943dfdae7a6c0e1c17520858c36e97d374ad6

Comment (dunn) [inline Formula/activemq.rb]: Oops, thanks!

Comment (DomT4) [inline Formula/activemq.rb]: No worries 😃

**Pull request closed without merging**